packages/nextra/src/client/components/search.tsx

```diff
@@ -238,17 +265,18 @@ export function Search({
         }
       >
         {error ? (
-          <div className="_h-full _flex _items-center _justify-center _gap-2 _px-8 _text-sm _text-red-500">
+          <>
             <InformationCircleIcon height="20" className="_shrink-0" />
-            {errorText}
-            <br />
-            {error.constructor.name}: {error.message}
-          </div>
+            <div className="_grid">
```

Comment:
Should we add `max-md:_content-start` to prevent this?

![image](https://github.com/user-attachments/assets/8fd548ab-21a7-46ae-9f85-13db8bf73631)

Reply:
Yeah, some regression